src/Jellyfin.Plugin.Dlna.Model/DeviceIdentification.cs

```diff
@@ -59,5 +59,5 @@ public class DeviceIdentification
     /// Gets or sets the headers.
     /// </summary>
     /// <value>The headers.</value>
-    public IReadOnlyList<HttpHeaderInfo> Headers { get; set; } = [];
+    public HttpHeaderInfo[] Headers { get; set; } = [];
```

Comment:
Please recheck if this is nessesary in the context of XML serialisation otherwise please revert

Reply:
Object is referenced by `DlnaDeviceProfile` which is used in the API.